infrasim/model.py

```diff
@@ -2141,6 +2145,10 @@ def init(self):
             racadm_obj.set_task_name("{}-racadm".format(self.__node_name))
             racadm_obj.set_log_path("/var/log/infrasim/{}/racadm.log".
                                     format(self.__node_name))
+
+            if "racadm_port" in self.__node:
```

Comment:
@JuneZhou19 , "racadm_port" is deprecated, now we are using "port" under "racadm".

Please fix this part, and below 5 uses in test.